### DISCUSSION TITLE
Question: how would i build the switch version

### DISCUSSION THREAD

**Discussion by arceuss** [FunkinCrew/Funkin#1493] [Q&A] [answered]
im trying to build it but it says i dont have limes switch thing or whatever
is there a homebrew version of it cuz by the looks of it you need a dev acc

**Comment by ninjamuffin99** [comment, marked as answer]
You would need the Lime-Switch development builds, which will build to official Nintendo Switch devkits. You can only get access to the lime-switch dev stuff, and get access to a devkit if you're an approved Nintendo Dev, who signed the NDA and has access to their SDK and all that!

Right now I don't even have that access!!!

**Comment by arceuss** [comment]
i have a acc but no switch access lol

**Comment by arceuss** [comment]
actually you can probably get in contact with someone at [here](https://www.ratalaikagames.com/services.php) to make eshop possible

**Comment by GStudiosX2** [comment]
https://developer.nintendo.com/tools

I found this site where you can get access to it but it requires alot of info idk why I wouldn't sign up for it but you can it's Ur choice
